google/cloud/vision/entity.py

```diff
@@ -55,9 +55,7 @@ def from_api_repr(cls, response):
         :rtype: :class:`~google.cloud.vision.entiy.EntityAnnotation`
         :returns: Instance of ``EntityAnnotation``.
         """
-        bounds = []
-        if 'boundingPoly' in response:
-            bounds = Bounds.from_api_repr(response['boundingPoly'])
+        bounds = Bounds.from_api_repr(response.get('boundingPoly', {}))
```

Comment:
It is desirable to have an "empty" `Bounds` instance in this case?  Wouldn't leaving `bounds` as `None` be a better marker?